data_steward/utils/participant_summary_requests.py

```diff
@@ -62,40 +71,168 @@ def get_access_token():
+    """
+    participant_records = []
+
+    # loop over participant summary records, insert participant data in same order as deactivated_participant_cols
+    for full_participant_record in api_data:
+        resource = full_participant_record.get('resource', {})
+        # loop over fields that exist in both resource_dict and columns and save key-value pairs
+        limited_participant_record = {
+            col: resource[col] for col in resource.keys() & columns
+        }
+        participant_records.append(limited_participant_record)
+
+    df = pandas.DataFrame.from_records(participant_records, columns=columns)
+
+    # Transforms participantId to an integer string
+    df['participantId'] = df['participantId'].apply(participant_id_to_int)
+
+    # Rename columns to be consistent with curation naming convention
+    bq_columns = {k: camel_to_snake_case(k) for k in columns}
+    df.rename(bq_columns, axis='columns', inplace=True)
+    df.rename(column_map, axis='columns', inplace=True)
+    return df
+
+
+def process_digital_health_data_to_json(api_data: List[Dict],
+                                        columns: List[str],
+                                        column_map: Dict) -> List[Dict]:
+    """
+    Converts digital health data retrieved from PS API to curation convention formatted json objects
+
+    :param api_data: digital_health_sharing_status data retrieved from PS API
+    :param columns: columns of interest
+    :param column_map: columns to be renamed as {old_name: new_name, ..}
+    :return: list of json objects with supplied columns and renamed as per column_map
+    """
+    participant_records = []
+
+    # loop over participant summary records, insert participant data in same order as deactivated_participant_cols
+    for full_participant_record in api_data:
+        resource = full_participant_record.get('resource', {})
+        # loop over fields that exist in both resource_dict and columns and save reformatted key-value pairs
+        limited_participant_record = {
+            camel_to_snake_case(col): resource[col]
+            for col in resource.keys() & columns
+        }
+
+        # re-map for columns that are to be mapped
+        for col in column_map & limited_participant_record.keys():
+            limited_participant_record[
+                column_map[col]] = limited_participant_record.pop(col)
+
+        # convert participant id to integer
+        limited_participant_record['person_id'] = participant_id_to_int(
+            limited_participant_record.get('person_id'))
+
+        # Extract all wearables data into a list of dicts
+        wearables_data = limited_participant_record.pop(
+            DIGITAL_HEALTH_SHARING_STATUS)
+
+        # loop over all wearables data
+        for wearable, latest_record in wearables_data.items():
+            # Set type of wearable
+            limited_participant_record['wearable'] = wearable
```

Comment:
This is intentional. RDR sends data by participant but if they are enrolled in multiple wearables, they need multiple records because of the way the digital_health_sharing_status table is set up.